include/flox/pkgdb/command.hh

```diff
@@ -21,7 +21,7 @@ namespace flox::pkgdb {
 /* -------------------------------------------------------------------------- */
 
 /** @brief Adds a single package database path to a state blob. */
-struct DbPathMixin : virtual public flox::NixState
```

Comment:
This change is generic cleanup.
I realized when editing this file that `DbPathMixin` does not actually require any members of `flox::NixState` and that the parent class was unnecessary.